Source/VMware.vSphereDSC/DSCResources/VMHostNetwork/VMHostVss/VMHostVss.ps1

```diff
@@ -188,10 +160,8 @@ class VMHostVss : VMHostBaseDSC {
             $vssConfigArgs.Add('Operation', 'remove')
         }
 
-        $vssConfig = New-VssConfig @vssConfigArgs
-        $vmHostNetworkSystem = Get-View -Server $this.Connection -Id $vmHost.ExtensionData.ConfigManager.NetworkSystem
         try {
-            Update-Network -NetworkSystem $vmHostNetworkSystem -Type 'VSS' -VssConfig $vssConfig
+            Update-Network -NetworkSystem $this.vmHostNetworkSystem -VssConfig $vssConfigArgs
```

Comment:
Here you should add the following parameter:

```
 -ErrorAction Stop
```

And add dot at the end of the message and also "Virtual" and "Switch" with capital letters.

Reply:
Changed